### DISCUSSION TITLE
Title naming of new notes

### DISCUSSION THREAD

**Discussion by spc789** [damionx7/Zettel-Notes-Documentation#106] [Feature Requests]
After a new note is made the standard file name can be set to `$$date$$.md`. 
However an empty note can show up with a title untitled. 
It could be good practice to indicated that this because it are empty notes, for example adding the filename in brackets along the untitled naming). Without that possibility one could end up editing in different files if one selects an "untitled" note randomly

**Comment by aamott** [comment]
Alternatively, empty notes could be removed? Or notes with only the default text for the repo/notebook.

**Comment by damionx7** [comment]
> After a new note is made the standard file name can be set to `$$date$$.md`. 
> However an empty note can show up with a title untitled. 
> It could be good practice to indicated that this because it are empty notes, for example adding the filename in brackets along the untitled naming). Without that possibility one could end up editing in different files if one selects an "untitled" note randomly

Can you give some examples of filename in brackets ?
